src/js/components/Filter/FilterForm.jsx

```diff
@@ -31,21 +33,18 @@ const FilterForm = ({
     },
   };
   // Create initialValues from filterFields as empty values
-  const initialValues = Object.keys(filterFields).reduce((acc, curr) => {
-    if (!acc[curr]) {
-      return {
-        ...acc,
-        [curr]: '',
-      };
-    }
-    return acc;
-  }, { ...defaultValues });
-  const [amountFilled, setAmountFilled] = useState(0);
+  let initialValues = Object.keys(filterFields)
+    .reduce((acc, key) => ({ ...acc, [`${key}`]: null }), {});
+  initialValues = { ...initialValues, ...defaultValues };
+
+  // Calculate which object's values are not empty
   const countFilled = (values) => {
-    // Calculate which object's values are not empty
-    setAmountFilled(Object.values(values).filter(value => !_.isEmpty(value)).length);
+    setAmountFilled(Object.values(values)
```

Comment:
the problem was that `_.isEmpety(...)` lodash function evaluates `boolean` values as `false`